ui/src/features/stage/promotion-details-modal.tsx

```diff
@@ -188,6 +234,22 @@ export const PromotionDetailsModal = ({
       onOk={hide}
       onCancel={hide}
       cancelButtonProps={{ hidden: true }}
+      footer={
+        canAbortPromotion(promotion) && (
+          <>
+            <Button onClick={hide} type='primary'>
+              Close
+            </Button>
+            <Button
+              danger
+              icon={<FontAwesomeIcon icon={faStopCircle} className='text-lg' />}
+              onClick={confirmAbortRequest}
+            >
+              Abort
+            </Button>
```

Comment:
I wonder if this place is the best. On the right side, next to the cancel button usually we have confirmation/main action button. I like the danger color, but WDYT about moving this button to the left side?

Reply:
Makes sense but everywhere else we have action buttons from right to left. So if we change it here, we have to change at lots of places